WordPress/Classes/ViewRelated/Stats/Shared Views/GhostViews/StatsGhostLineChartCell.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+import WordPressShared
+
+class StatsGhostLineChartCell: StatsGhostBaseCell, NibLoadable {
+    @IBOutlet private weak var lineChart: StatsGhostLineChartView!
+}
```

Comment:
nitpick: Using an XIB means the Design System can't be used for padding. If you'd like to use it, you could delete the XIB and created the cell in code:

```suggestion

import DesignSystem

class StatsGhostLineChartCell: StatsGhostBaseCell {

    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
        let lineChart = StatsGhostLineChartView()
        super.init(style: style, reuseIdentifier: reuseIdentifier)

        headingLabel.isGhostableDisabled = true

        lineChart.translatesAutoresizingMaskIntoConstraints = false
        contentView.addSubview(lineChart)

        topConstraint = lineChart.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 0)
        topConstraint?.isActive = true

        NSLayoutConstraint.activate([
            lineChart.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .DS.Padding.double),
            contentView.trailingAnchor.constraint(equalTo: lineChart.trailingAnchor, constant: .DS.Padding.double),
            contentView.bottomAnchor.constraint(equalTo: lineChart.bottomAnchor, constant: .DS.Padding.single),
            lineChart.heightAnchor.constraint(equalToConstant: 190),
        ])
    }

    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
}
```

It can then be initialized with:

```
        return ImmuTableCell.class(StatsGhostLineChartCell.self)
```

I tested this (different orientations, dark/light mode, etc), and it works well.  

The same idea applies to StatsGhostSingleValueCell.xib and StatsGhostTopCell.xib. Although there's a more complicated layout to create in code, I wonder if it's more maintainable (in addition to the benefits of leveraging the Design System).

Reply:
Great suggestion. Choosing to use Xib was a lazy approach but not modern, easy to reuse, or integratable with the Design System. I'll apply your suggestions.